.circleci/scripts/rerun-ci-workflow-from-failed.ts

```diff
@@ -0,0 +1,207 @@
+const CIRCLE_TOKEN = process.env.API_V2_TOKEN;
+
+interface Actor {
+  login: string;
+  avatar_url: string | null;
+}
+
+interface Trigger {
+  received_at: string;
+  type: string;
+  actor: Actor;
+}
+
+interface VCS {
+  origin_repository_url: string;
+  target_repository_url: string;
+  revision: string;
+  provider_name: string;
+  branch: string;
+}
+
+interface WorkflowItem {
+  id: string;
+  errors: string[];
+  project_slug: string;
+  updated_at: string;
+  number: number;
+  state: string;
+  created_at: string;
+  trigger: Trigger;
+  vcs: VCS;
+}
+
+interface CircleCIResponse {
+  next_page_token: string | null;
+  items: WorkflowItem[];
+}
+
+interface WorkflowStatusItem {
+  pipeline_id: string;
+  id: string;
+  name: string;
+  project_slug: string;
+  tag?: string;
+  status: string;
+  started_by: string;
+  pipeline_number: number;
+  created_at: string;
+  stopped_at: string;
+}
+
+interface WorkflowStatusResponse {
+  next_page_token: string | null;
+  items: WorkflowStatusItem[];
+}
+
+/**
+ * Fetches the last 20 CircleCI workflows for 'develop' branch.
+ * Note: the API returns the first 20 workflows by default.
+ * If we wanted to get older workflows, we would need to use the 'page-token' we would get in the first response
+ * and perform a subsequent request with the 'page-token' parameter.
+ * This seems unnecessary as of today, as the amount of daily PRs merged to develop is not that high.
```

Comment:
Good decision. Easier to run this multiple times throughout the day, rather than support paging through more runs.